src/test/Number.test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { shallow } from 'enzyme'
+
+import Number from '../components/widgets/Number'
+
+describe( 'Number widget component', () => {
+  const statistic = '220.00'
+  const goal = '200'
+  const progress = ( statistic / goal ) * 100
+  const roundedProgress = parseFloat( progress ).toFixed( 2 )
+
+  const widgetData = {
```

Comment:
Create a folder for dummy csv widgets in reducers/widgets/dummy-data and make a file and put this widgetData in it. That way folks can see your widget easily by putting it in the default state of the widgets reducer.